documentation/modules/exploit/windows/http/manageengine_adselfservice_plus_cve_2021_40539.md

```diff
@@ -0,0 +1,89 @@
+## Vulnerable Application
+
+### Setup
+
+Follow the [Getting Started] guide. You do not need AD, and you can skip licensing.
+
+1. Download [ManageEngine_ADSelfService_Plus_64bit.exe]
+1. Run the installer
+1. Start the server
+1. Hax
+
+[Getting Started]: https://www.manageengine.com/products/self-service-password/help/admin-guide/gettingstarted/getting-started.html
+[ManageEngine_ADSelfService_Plus_64bit.exe]: https://archives2.manageengine.com/self-service-password/6113/ManageEngine_ADSelfService_Plus_64bit.exe
+
+## Verification Steps
+
+Follow [Setup](#setup) and [Scenarios](#scenarios).
+
+## Options
+
+### `TARGETURI`
```

Comment:
Quoting this as it is here prevents it from rendering as H3.

```suggestion
### TARGETURI
```

Reply:
Yeah, that's a [Redcarpet](https://github.com/vmg/redcarpet) bug. GitHub renders it just fine. I guess I'll revert it for now.

Reply:
Yeah, that's a bug in our local renderer's styling. GitHub renders it just fine.

Reply:
It's [this thing](https://github.com/rapid7/metasploit-framework/blob/master/data/markdown_doc/markdown.css)'s fault. :)

Reply:
I was able to fix it, but I don't know CSS well enough to PR it. :-)